corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -998,8 +1033,8 @@ def properly_wrap_sync_log(doc, synclog_sql=None):
 
 
 def get_sync_log_class_by_format(format):
-    if format == LOG_FORMAT_LEGACY:
-        raise SyncLogAssertionError('Legacy type not supported')
+    if format not in (LOG_FORMAT_SIMPLIFIED, LOG_FORMAT_LIVEQUERY):
+        raise SyncLogAssertionError('Unrecognized synclog format')
```

Comment:
Is the format set by HQ? If so, it seems like a place where we could get rid of a bit of indirection by replacing all instances with `SimplifiedSyncLog`

Reply:
6c3d4170acd